skrub/dataframe/tests/test_namespace.py

```diff
@@ -0,0 +1,41 @@
+import pandas as pd
```

Comment:
I think you need an `__init__.py` in this directory (at least with the current approach of having test modules be part of the package), otherwise for me locally pytest fails and complains it cannot import `skrub.dataframe`